utils/sub8_diagnostics/scripts/thruster_spinner.py

```diff
@@ -31,7 +31,10 @@ def ports_from_layout(layout):
             for name in thruster_names:
```

Comment:
Not relevant to the goal of this PR, but in testing with Ken I found it would be helpful to be able to go reverse in thruster_spinner. I think This just means you have to change the limits on the effort command from [0.0, 1.0] to [-1.0, 1.0]